polymorphic/admin/inlines.py

```diff
@@ -53,7 +53,8 @@ class PolymorphicInlineModelAdmin(InlineModelAdmin):
 
     #: Inlines for all model sub types that can be displayed in this inline.
     #: Each row is a :class:`PolymorphicInlineModelAdmin.Child`
```

Comment:
The semantics of `None` will need to be documented here.

I also don't think this should be automatic by default – it could mean that upgrading Polymorphic in a project suddenly exposes things in the admin that one might not want to expose.

Reply:
Why don't you think they should be exposed? The current alternative is that code crash on them.

Reply:
Sorry, I'm not following. That the code crashes on... what exactly?

Reply:
They Crash if there are no child classes defined.